test/qunit/utils/TestUtils.js

```diff
@@ -166,6 +186,60 @@ sap.ui.define(["sap/base/util/merge"], function (merge) {
 		return mUrlParameters;
 	};
 
+	/**
+	 * Creates a new FHIRModel instance
+	 *
+	 * @protected
+	 * @param {string} sServiceBaseUrl The FHIR service base url
+	 * @param {object} mParameters The model parameters
+	 * @returns {sap.fhir.model.r4.FHIRModel} The newly created FHIRModel instance
+	 * @since 0.0.2
+	 */
+	TestUtils.createFHIRModel = function (sServiceBaseUrl, mParameters) {
+		return new FHIRModel(sServiceBaseUrl, mParameters);
+	};
```

Comment:
This method is currently only overhead, if their would be any other logic, e.g parsing parameters or the service url I would understand the purpose of the method